server/controllers/articlecontroller.js

```diff
@@ -2,7 +2,7 @@ import dotenv from 'dotenv';
 // eslint-disable-next-line import/no-unresolved
```

Comment:
you are still ignoring eslint rules

Reply:
I will try  to solve them as much as I can